src/current/v23.1/monitor-and-debug-changefeeds.md

```diff
@@ -157,6 +157,39 @@ changefeed_emitted_bytes{scope="vehicles"} 183557
 `backfill_pending_ranges` | Number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) in an ongoing backfill that are yet to be fully emitted. | Ranges
 `message_size_hist` | Distribution in the size of emitted messages. | Bytes
 
+### Monitoing and measuring changefeed latency
+
+Changefeeds can encounter latency in **events** emitting. This latency is the total time CockroachDB takes to:
+
+- Commit writes to the database.
+- Encode [changefeed messages]({% link {{ page.version.version }}/changefeed-messages.md %}).
+- Deliver the message to the [sink]({% link {{ page.version.version }}/changefeed-sinks.md %}).
+
+There are a couple of ways to measure if changefeeds are encountering latency or falling behind:
+
+- [Event latency](#event-latency): Measure the difference between an event's MVCC timestamp and when it is put into the memory buffer or acknowledged at the sink.
+- [Lagging ranges](#lagging-ranges): Track the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#range) that are behind in a changefeed.
+
+#### Event latency
+
+To monitor for changefeeds encountering latency in how events are emitting, track the following metrics:
+
+- `admit_latency`: The difference between the event's MVCC timestamp and the time the event is put into the memory buffer.
+- `commit_latency`: The difference between the event's MVCC timestamp and the time it is acknowledged by the [downstream sink]({% link {{ page.version.version }}/changefeed-sinks.md %}). If the sink is batching events, the difference is between the oldest event and when the acknowledgment is recorded.
+
```

Comment:
We should indicate that these metrics are *not* updated during backfill events.
(Why? Because you might be backfilling (i.e doing initial scan) over old data -- and the fact that it's been "10 months" since the wrote was written does not mean that there is latency).  Backfills also may happen during schema changes -- and those are excluded as well (similar reason: but basically, backfills could take hours).

Reply:
I added a note to the section to refer to this — can you check out the wording here: https://deploy-preview-18185--cockroachdb-docs.netlify.app/docs/v23.2/monitor-and-debug-changefeeds.html#event-latency